Improving map paths

boxxxie opened the pull request.

boxxxie committed:
Fix Map.key_paths to work with structs

kenips commented on lib/kitchen_sink/assert.ex:
```diff
@@ -0,0 +1,67 @@
+
+  import ExUnit.Assertions
+
+  def assertish(expected, actual, epsilon, msg \\ "") when is_map(actual) and is_map(expected) do
```
Need proper docs for this

boxxxie replied:
fixed

boxxxie committed:
Add Assert.assertish/4

assertish will let you do delta comparisons on objects!
if there are errors, it will print out each error path and comparison value
